mistletoe/latex_renderer.py

```diff
@@ -57,15 +57,21 @@ def render_image(self, token):
         self.packages['graphicx'] = []
         return '\n\\includegraphics{{{}}}\n'.format(token.src)
 
+    @staticmethod
+    def render_url(token):
+        return token.target.replace('%', '\\%') \
+                           .replace('#', '\\#')
```

Comment:
Can we rename it and unify this with already existing solution as seen at `HTMLRenderer.escape_url()`?

* it is a helper method, so it belongs to the end of the class
* the method should accept a string. This way, we can reuse it on another place.

Reply:
Done in 97bfce11

Reply:
OK.